megamek/src/megamek/common/Mech.java

```diff
@@ -6101,11 +6045,7 @@ public int getActiveSinksNextRound() {
      * @return Returns the autoEject.
      */
     public boolean isAutoEject() {
-        boolean hasEjectSeat = true;
-        if (getCockpitType() == COCKPIT_TORSO_MOUNTED
-                || hasQuirk(OptionsConstants.QUIRK_NEG_NO_EJECT)) {
-            hasEjectSeat = false;
-        }
+        boolean hasEjectSeat = getCockpitType() != COCKPIT_TORSO_MOUNTED && !hasQuirk(OptionsConstants.QUIRK_NEG_NO_EJECT);
```

Comment:
No point undoing it since you already did the work, but, in general, when submitting PRs, my personal preference is to avoid changes like this that have no bearing on the functionality being implemented; mostly it serves to worsen the signal-to-noise ratio, making the actual relevant changes harder to separate and review. At least for me with my old person brain.

Reply:
im not sure how this got in here. probably hit a 'fix it' button in the IDE accidentally